app/src/js/utils/table-config/input.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+import React from 'react';
+import { inputLink } from '../format';
+import Dropdown from '../../components/DropDown/simple-dropdown';
+import { Link } from 'react-router-dom';
+
+export const tableColumns = [
+  {
+    Header: 'Question ID',
+    accessor: (row) => row.question_id,
+    Cell: row => inputLink(row.row.original.question_id, row.row.original.control_id),
```

Comment:
This is a little misleading that the link is on the question ID, but the link takes you to the input. What do you think about adding the link on the control id instead?
![Screenshot 2024-11-15 085405](https://github.com/user-attachments/assets/d1416526-8703-4ddb-960e-e4ff4b0d0cfd)